flexmeasures/data/services/users.py

```diff
@@ -27,9 +27,7 @@ class InvalidFlexMeasuresUser(Exception):
 
 def get_user(id: str) -> User:
     """Get a user, raise if not found."""
-    user: User = db.session.execute(
-        select(User).filter_by(id=int(id))
-    ).scalar_one_or_none()
+    user: User = db.session.execute(User, int(id))
```

Comment:
The issue causing the failing tests looks like it is related to a User instance, and of course the tests started to fail with this big commit, which lead me to double check this edit. I believe this is supposed to be `db.session.get` rather than `db.session.execute`.